pkg/dceerr/error.go

```diff
@@ -0,0 +1,208 @@
+package dceerr
```

Comment:
it feels a little funny having the product name inside the pkg name.
Maybe just some lingering bad feeling around the "Redbox" refactor 😉 

Is the idea to avoid naming conflicts with the stdlib error package?

Reply:
good question.  I could switch to `errors` but it conflicts with the global `errors` package.   I tried not to do that.  I agree I don't like the dce portion of it.  I'm game for changing it.

Reply:
I agree. can we remove dce from package name

Reply:
sure. I don't really have a better name, though, if we don't want to use  `errors`. so it's ok with me